bower.json

```diff
@@ -37,6 +37,7 @@
     "purescript-assert": "master",
     "purescript-console": "master",
     "purescript-math": "master",
-    "purescript-minibench": "master"
+    "purescript-minibench": "master",
+    "purescript-quickcheck": "master"
```

Comment:
After install `quickcheck` as a dev dependency, I can't build the project. Getting lots of "DuplicateModule" errors, such as:
```
Module Data.List has been defined multiple times
```

Reply:
I'd assume that's because of a cyclical dependency:
https://github.com/purescript/purescript-quickcheck/blob/082f1f6681b6fc538e719622e35728d4de6d76e8/bower.json#L37

Reply:
Yep. I’m not aware of any good way around this unfortunately. I think if we put all of the core libraries into one big monorepo or even one big library we might be able to work around dependency issues like this, but neither of those are workable in the short term anyway. We will just have to do without for now. You could set up a separate project to perform these tests for peace of mind if that would help.

Reply:
Quickcheck removed